### ISSUE TITLE
Paper List looks too long, maybe folding part of the paper list ?

### ISSUE THREAD

**Issue by lwaekfjlk** [ulab-uiuc/AGI-survey#5]
Currently the paper list looks too long, maybe we can fold some related but not important paper to make the repo look cleaner?

**Comment by Kunlun-Zhu** [comment]
good point, we shall design better ways to make the paper list look cleaner.

**Comment by lwaekfjlk** [comment]
I suggest we just list a few important papers and make the rest folded in README.

**Comment by James4Ever0** [comment]
#9
